zipkin-server/pom.xml

```diff
@@ -225,6 +225,17 @@
       <version>${micrometer.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>com.netflix.concurrency-limits</groupId>
+      <artifactId>concurrency-limits-core</artifactId>
+      <version>${netflix.concurrency.limits.version}</version>
+    </dependency>
+    <dependency>
```

Comment:
We have a convention of directly depending on any artifact that is used.  Since I added code that directly used this I added the dependency.  Otherwise, yes, I believe it was being pulled transitively.  But that causes problems if your dependencies change ever.

However, that's just food for thought.  If you still prefer, I can definitely remove this?

Reply:
no need